lib/src/components/NxRadio/NxRadio.scss

```diff
@@ -10,52 +10,66 @@
 @import '../../scss-shared/nx-input-helpers';
 
 .nx-radio {
-  $circle-width: 16px;
-  $label-margin: 8px;
-  $line-height: 20px;
-
   @include container-horizontal;
   @include regular();
   @include focusable-toggle-input(".nx-radio__outer-circle");
 
   display: inline-block;
-  font-size: 14px;
-  line-height: $line-height;
+  margin-bottom: $nx-spacing-xs;
   position: relative;
   white-space: nowrap;
 
-  &:hover:not(.nx-radio--disabled) {
-    cursor: pointer;
+  &:hover {
+    .nx-radio__outer-circle {
+      stroke: #2c2c2c;
+    }
   }
+}
 
+.nx-radio--disabled {
   .nx-radio__circle {
-    color: $nx-text-color-light;
-    cursor: pointer;
-    height: $line-height;
-    margin-right: $label-margin;
-    width: $circle-width;
-    vertical-align: top;
+    cursor: default;
+  }
 
-    &:last-child {
-      // due to specificity issues the `container-horizontal` mixin on nx-radio isn't successfully setting this
-      margin-right: 0;
+  &, &:hover {
+    .nx-radio__outer-circle {
+      fill: #e9e9e9;
+      stroke: #949494;
     }
   }
+}
 
-  .nx-radio__outer-circle {
-    stroke: currentColor;
-  }
+.nx-radio__circle {
+  cursor: pointer;
+  height: 24px;
+  margin-right: $nx-spacing-md;
+  width: 16px;
+  vertical-align: top;
 
-  .nx-radio__inner-circle {
-    fill: currentColor;
+  &:last-child {
+    // due to specificity issues the `container-horizontal` mixin on nx-radio isn't successfully setting this
+    margin-right: 0;
```

Comment:
Doesn't appear to be needed anymore

Reply:
> Doesn't appear to be needed anymore

Removed here: https://github.com/sonatype/sonatype-react-shared-components/pull/86/commits/044c3d73d644c420c1a85954d25fd38f2c3829c1